iothub/device/tests/Authentication/IotHubConnectionStringBuilderTests.cs

```diff
@@ -145,27 +226,58 @@ public void IotHubConnectionStringBuilder_ParamConnectionString_ParsesGatewayHos
         }
 
         [TestMethod]
-        [ExpectedException(typeof(FormatException))]
+        public void IotHubConnectionStringBuilder_ParamConnectionString_Equals()
+        {
+            var connectionString = $"HostName={HostName};DeviceId={DeviceId};ModuleId={ModuleId};SharedAccessKeyName={SharedAccessKeyName};SharedAccessKey={SharedAccessKey}";
+            var iotHubConnectionCredentials = new IotHubConnectionCredentials(connectionString);
+            var iotHubConnectionCredentials_copy = new IotHubConnectionCredentials(connectionString);
+            bool equal = iotHubConnectionCredentials.Equals(iotHubConnectionCredentials_copy);
+            equal.Should().BeTrue();
+        }
+
+        [TestMethod]
         public void IotHubConnectionStringBuilder_ParamConnectionString_MissingHostName_Throws()
         {
             var connectionString = $"DeviceId={DeviceId};SharedAccessKey={SharedAccessKey}";
-            var iotHubConnectionCredentials = new IotHubConnectionCredentials(connectionString);
+            Action act = () => _ = new IotHubConnectionCredentials(connectionString);
+            act.Should().Throw<FormatException>();
```

Comment:
How is this different from the test for `IotHubConnectionStringBuilder_ParamConnectionString_ValidateHostName()`?

Reply:
removed duplicate test.